content/hi/docs/concepts/overview/working-with-objects/owners-dependents.md

```diff
@@ -0,0 +1,88 @@
+---
+title: मालिक और आश्रित
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+
+कुबेरनेट्स में, कुछ {{< glossary_tooltip text="objects" term_id="object" >}} हैं
+*owners* अन्य वस्तुओं का. उदाहरण के लिए,
+{{<glossary_tooltip text="ReplicaSet" term_id="replica-set">}} है owner
+पॉड्स के एक सेट का। ये स्वामित्व वाली वस्तुएँ हैं *dependents* उनके owner.

```

Comment:
पॉड्स के एक सेट का। ये स्वामित्व वाली वस्तुएँ अपने owner पर *dependents* हैं।
I think this sentence looks more meaning full WDYT?

Reply:
Yes, I also think!